online-shop/src/app/components/presentational/products-form-view/products-form-view.component.html

```diff
@@ -0,0 +1,15 @@
+<body style="padding-left: 2rem">
```

Comment:
Since this is the product form view, the naming is a bit confusing, since you have the add functionality here. Please check what I have mentioned above, with using 1 single component for add&edit